### ISSUE TITLE
crux-mir test suite failures

### ISSUE THREAD

**Issue by robdockins** [GaloisInc/crucible#578]
I can't figure out how to run the `crux-mir` test suite locally.  When I run it, i get a cavalcade of similar errors.

```
Test suite test: RUNNING...
crux-mir
  crux concrete

      impl
        simple:                  FAIL (0.25s)
          Compiling and running oracle program (0.21s)
          Oracle output: 42                    (0.04s)
            user error (Error 101 while running mir-json on test/conc_eval/impl/simple.rs)
        self_mut:                FAIL (0.28s)
          Compiling and running oracle program (0.23s)
          Oracle output: 42                    (0.04s)
            user error (Error 101 while running mir-json on test/conc_eval/impl/self_mut.rs)
        self:                    FAIL (0.27s)
          Compiling and running oracle program (0.22s)
          Oracle output: 42                    (0.05s)
            user error (Error 101 while running mir-json on test/conc_eval/impl/self.rs)
```

This goes on for a long time, with a lot of `Error 101`.  I don't know how to fix it.

**Comment by voidc** [comment]
I get the same errors. To me it seems like mir-json fails during the "linking" step.

**Comment by spernsteiner** [comment]
Please try running:
```sh
cabal v2-run -- crux-mir -d3 test/conc_eval/impl/simple.rs
```
This will pass through any errors reported by `mir-json`.

If it reports ``can't find crate for `core` ``, then please re-run `./translate_libs.sh` and try again.

If it reports that `core` was compiled by a different version of `rustc`, then please check `rustc --version`.  If it's not `rustc 1.44.0-nightly (38114ff16 2020-03-21)` (the version required by crux-mir), then try setting the Rust toolchain to the appropriate nightly and re-running `translate_libs.sh`:
```sh
rustup default nightly-2020-03-22
./translate_libs.sh
```
It should be safe to set the toolchain back to its previous value afterward.

**Comment by robdockins** [comment]
Some combination of `rustup` and `translate_libs.sh` commands got the `Error 101` messages to go away.  I'm still seeing 10 unexpected test failures, but it seems like they are just related to file name differences.

**Comment by spernsteiner** [comment]
Fixed in #628
